addons/htpasswd-identity-provider/htpasswd-identity-provider.addon

```diff
@@ -0,0 +1,30 @@
+# Name: htpasswd-identity-provider
+# Description: Configures minishift to use HTPasswdIdentityProvider
+# Url: https://docs.openshift.org/3.9/install_config/configuring_authentication.html#HTPasswdPasswordIdentityProvider
+# Var-Defaults: USERNAME=developer,USER_PASSWORD=superS3cret,MINISHIFT_DATA_HOME=/var/lib/minishift,CONFIG_LOCATION=openshift.local.config/master,MASTER_CONFIG_FILE=master-config.yaml,ORIGIN_DATA_HOME=/var/lib/origin
```

Comment:
@minishift/minishift-dev I have added the default password also as it will be helpful if user wants to enable the add-on and it would run automatically after minishift start.